spec/models/organization_spec.rb

```diff
@@ -8,4 +8,43 @@
 
   it_should_behave_like 'notable'
 
+  it 'should set and get custom values' do
```

Comment:
Part of the reason we define multi-model concern specs using shared example groups is so that we don't need to duplicate all this work in every model. If the worry is testing that specific features work, so that you can make sure the "plumbing" is all good for utilizing these for actual use cases, that'll be better handled in feature/integration specs once there's a UI for a user utilizing these features. I would factor out the parts of this that are mostly about whether the concerns/modules are working properly into a shared example group.